analytics/views.py

```diff
@@ -1559,25 +1560,25 @@ def format_date_for_activity_reports(date: Optional[datetime]) -> str:
 
 def user_activity_link(email: str) -> mark_safe:
     url = reverse(get_user_activity, kwargs=dict(email=email))
-    email_link = f'<a href="{url}">{email}</a>'
+    email_link = escape(f'<a href="{url}">{email}</a>')
```

Comment:
That’s not how escaping works.

```suggestion
    email_link = f'<a href="{escape(url)}">{escape(email)}</a>'
```

Reply:
Thanks!